projects/lua/lua_dump_test-lf.toml

```diff
@@ -0,0 +1,27 @@
+# Copyright 2025 Sergey Bronnikov
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+[sydr]
+args = "--wait-jobs -s 90 -j2"
+target = "/lua_dump_sydr @@"
+jobs = 2
+
+[libfuzzer]
+path = "/lua_dump_test"
+args = "-dict=/lua_dump.dict /corpus_lua_dump"
```

Comment:
lua_dump.dict is missing, add lua_dump_test.dict to ua-c-api-corpus

Reply:
Fixed by https://github.com/ligurio/lua-c-api-corpus/commit/ec8ddc95d4c49c440203a86ad031898b554e873c.